webpack.config.js

```diff
@@ -0,0 +1,36 @@
+const defaultsDeep = require('lodash.defaultsdeep');
+const path = require('path');
+const webpack = require('webpack');
+
+
+const base = {
```

Comment:
Also, since we're only building one thing, this can be collapsed into a single export (no base that gets extended).